File: lecture1/psets/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("enter Positive Number: ")
    }
    while (height < 1 || height > 8)

    for (int i = 0; i < height; i++)
    {
        printf("#");
        for (int j = 0; j < height; j++)
        {
            printf("#");
            printf("\n");
        }

    printf("\n");
    }

}

File: lecture6/pset/mario-less/mario.py
from cs50 import get_int
def main():
    height = get_height()
    for i in range (height):
        print ("#")



#function to get hight
def get_height():
    while True:
        #get the user input while its between 1 and 8
        n = get_int()
        if n > 0 or n <= 8:
            return n

main()

